docker/gcb/base-images.yaml

```diff
@@ -1,57 +0,0 @@
-# Copyright 2020 Google LLC
```

Comment:
We use base-images.yaml to build base-image everytime this master is committed to. So we can't get rid of it.
Can you come up with a way to generate this file outside of an experiment (maybe if invoking generate_cloudbuild` from the commandline`) and then keep it in the repo?
If you follow my suggestions about doing the substitution while generating this file, just use gcr.io/fuzzbench for `REPO` and `test-experiment` for `experiment`. This is OK because only the official fuzzbench project will need this file.

Reply:
`main()` does that now, so should be callable from command line to write the base-images.yaml file.

Reply:
So please do that and commit the file.

Reply:
@lszekeres